src/name/bizna/ocarmsim/FakeMachine.java

```diff
@@ -17,27 +19,28 @@
 import li.cil.oc.api.network.Message;
 import li.cil.oc.api.network.Network;
 import li.cil.oc.api.network.Node;
+import name.bizna.jarm.Debugger;
+import name.bizna.ocarmsim.components.SimComponent;
+import net.minecraft.nbt.NBTTagCompound;
 
 public class FakeMachine implements Machine {
-	
-	public static String computerAddress = "e521fbeb-b67d-434e-9295-40c31361e6f9";
```

Comment:
In my initial review, I missed the part where you made the machine UUID dynamic, and assumed I had simply forgotten to implement `FakeMachine.address()`. You can ignore this comment.